package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.1.3",
   "description": "A namespaced stylish logger primarily for the browser",
   "main": "index.js",
```

Comment:
I thought the main entry point was going to stay as the browser? Or does SSG support the browser field?